app/js/index.js

```diff
@@ -110,11 +111,11 @@ angular.module('index-app', ['firebase'])
 							team: ['null']
 						});
 					}
+					$scope.username = usersArray.$getRecord(user.uid).name;
```

Comment:
sorry I saw that comment after I pull request second time.
I changed username to email then.